parm/config/config.base

```diff
@@ -31,5 +31,5 @@ export IC_OFFSET=${IC_OFFSET:-"3"}
 export LBC_PREFIX=${LBC_PREFIX:-"GFS"}
 export LBC_OFFSET=${LBC_OFFSET:-"6"}
 export LBC_INTERVAL=${LBC_INTERVAL:-"1"}
-export LBC_LENGTH=$((FCST_LENGTH+6)) # lbc runs every 6 hours # this variable means how many hours going forward from the LBC CDATE
+export LBC_LENGTH=${LBC_LENGTH:-"6"} # lbc runs every 6 hours # this variable means how many hours going forward from the LBC CDATE
```

Comment:
Each time we generate LBCs, we would like to cover 6h more. For example, 00z lbc would like to cover up to forecasts launched at up to 06z, that's why we initially set `LBC_LENGTH=$((FCST_LENGTH+6))`

Reply:
The LBC_LENGTH need to be decided based on the application. Many cases do not have fixed relation with the FCST_LENGTH, like ensemble da and forecast, or use RAP/RRFS as boundary input.

Reply:
Make sense. But I think you just removed the sourcing of `config.base`